scripts/docs/data_files/vcf_species_list_112.json

```diff
@@ -0,0 +1,131 @@
+      "count" : "<span class=\"vdoc_var_count vdoc_thousand\" title=\"Over 37,000 variants\">37 K</span>",
+      "name" : "Collared flycatcher",
+      "label" : "Ficedula albicollis"
+   },
+   "macaca_mulatta" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_million_1\" title=\"Over 50 million variants\">50 M</span>",
+      "name" : "Macaque",
+      "label" : "Macaca mulatta"
+   },
+   "pan_troglodytes" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_million_1\" title=\"Over 22 million variants\">22 M</span>",
+      "name" : "Chimpanzee",
+      "label" : "Pan troglodytes"
+   },
+   "coturnix_japonica" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_thousand\" title=\"Over 8,000 variants\">8 K</span>",
+      "name" : "Japanese quail",
+      "label" : "Coturnix japonica"
+   },
+   "bos_grunniens" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_thousand\" title=\"Over 64,000 variants\">64 K</span>",
+      "name" : "Domestic yak",
+      "label" : "Bos grunniens"
+   },
+   "anas_platyrhynchos" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_million_1\" title=\"Over 29 million variants\">29 M</span>",
+      "name" : "Mallard",
+      "label" : "Anas platyrhynchos"
+   },
+   "macaca_fascicularis" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_million_2\" title=\"Over 1.6 million variants\">1.6 M</span>",
+      "name" : "Crab-eating macaque",
+      "label" : "Macaca fascicularis"
+   },
+   "oreochromis_niloticus" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_thousand\" title=\"Over 63,000 variants\">63 K</span>",
+      "name" : "Nile tilapia",
+      "label" : "Oreochromis niloticus"
+   },
+   "oncorhynchus_mykiss" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_thousand\" title=\"Over 33,000 variants\">33 K</span>",
+      "name" : "Rainbow trout",
+      "label" : "Oncorhynchus mykiss"
+   },
+   "cavia_porcellus" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_thousand\" title=\"Over 844,000 variants\">844 K</span>",
+      "name" : "Guinea Pig",
+      "label" : "Cavia porcellus"
+   },
+   "taeniopygia_guttata" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_million_2\" title=\"Over 1.5 million variants\">1.5 M</span>",
+      "name" : "Zebra finch",
+      "label" : "Taeniopygia guttata"
+   },
+   "meleagris_gallopavo" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_thousand\" title=\"Over 5,000 variants\">5 K</span>",
+      "name" : "Turkey",
+      "label" : "Meleagris gallopavo"
+   },
+   "seriola_dumerili" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_million_2\" title=\"Over 9 million variants\">9 M</span>",
+      "name" : "Greater amberjack",
+      "label" : "Seriola dumerili"
+   },
+   "salmo_salar" : {
+      "count" : "<span class=\"vdoc_var_count vdoc_million_1\" title=\"Over 15 million variants\">15 M</span>",
+      "name" : "Atlantic salmon",
+      "label" : "Salmo salar",
+      "genotype" : 1
```

Comment:
Hi @jamie-m-a , thank you for the PR. 

The count is extracted from EVA study instead of PRJEB34225. Is "genotype" still 1?
For species having multiple sources, do we want to report counts from both the sources?

Reply:
Created a ticket: [ENSVAR-6289](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-6289)